posthog/models/feature_flag/feature_flag.py

```diff
@@ -297,6 +297,31 @@ def get_cohort_ids(
 
         return list(cohort_ids)
 
+    def scheduled_changes_dispatcher(self, payload):
+        from posthog.api.feature_flag import FeatureFlagSerializer
+
+        if "operation" not in payload or "value" not in payload:
+            raise Exception("Invalid payload")
+
+        context = {
+            "request": {"user": self.created_by},
+            "team_id": self.team_id,
+        }
+        serializer_data = {}
+
+        if payload["operation"] == "add_release_condition":
+            existing_groups = self.filters.get("groups", [])
```

Comment:
again, don't directly access filters, use `self.get_filters()` because that's where the backwards compatibility happens